### PR TITLE
WINDUP-895 moving hint detail from below row to right cell

### PR DESCRIPTION
**Pull request by mareknovotny**
I tried to move it from table row to nest table cell to the right of Show Hint column.
When showing it will add colspan="5" to the parent table cell to be wider for new column with hint details.

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1408/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/782/<h2>Build result: FAILURE</span></h2>[...truncated 65670 lines...][JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/classes/META-INF/resources/windup-victims-2.5.0-SNAPSHOT.dot to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT.dot[JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/windup-victims-2.5.0-SNAPSHOT-forge-addon.jar to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT-forge-addon.jarchannel stoppedWeld SE container org.jboss.windup.ast:windup-java-ast shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-archives shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-tattletale shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-groovy shut down by shutdown hookArchiving artifactsWeld SE container org.jboss.windup:windup-tooling shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-project shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-xml shut down by shutdown hookWeld SE container org.jboss.forge.addon:convert shut down by shutdown hookWeld SE container org.jboss.windup.exec:windup-exec shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-base shut down by shutdown hookWeld SE container org.jboss.windup.ui:windup-ui shut down by shutdown hookWeld SE container org.jboss.forge.addon:configuration shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-xml shut down by shutdown hookWeld SE container org.jboss.forge.addon:ui shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-ee shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config shut down by shutdown hookWeld SE container org.jboss.windup.reporting:windup-reporting shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java shut down by shutdown hookIRC notifier plugin: Sending notification to: #windupSetting status of 331bcc9253ba541b760d64dc6685fcef79f5339c to FAILURE with url http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/782/ and message: 'Build finished. 5 tests run, 0 skipped, 0 failed.'

**Comment by jsight** [comment]
Doesn't this just make cell #1 extremely large? Also, it looks like the 5 cell width is being placed on the wrong TD element.

**Comment by mareknovotny** [comment]
@jsight I think the growing of the first table column is wanted as hint text could be formatted with source code and I saw it pretty difficult to read if wrapped.
Ad `colspan="5"` I realized that it is leftover from my experiments even I described it like wanted in PR description. I experimented with adding row instead of just table column for showing hint, but it was messed up due sorting so I went with the table column addition at the end and it stayed in code, but it useless for the adding table column.

**Comment by mareknovotny** [comment]
Hidden hint
screenshot from 2016-02-01 08-40-54

Shown hint
screenshot from 2016-02-01 08-41-01

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1411/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/785/<h2>Build result: FAILURE</span></h2>[...truncated 65670 lines...][JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/classes/META-INF/resources/windup-victims-2.5.0-SNAPSHOT.dot to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT.dot[JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/windup-victims-2.5.0-SNAPSHOT-forge-addon.jar to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT-forge-addon.jarchannel stoppedWeld SE container org.jboss.windup.rules.apps:windup-rules-tattletale shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-xml shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-groovy shut down by shutdown hookWeld SE container org.jboss.forge.addon:convert shut down by shutdown hookWeld SE container org.jboss.windup.ast:windup-java-ast shut down by shutdown hookWeld SE container org.jboss.windup.ui:windup-ui shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-archives shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-project shut down by shutdown hookArchiving artifactsWeld SE container org.jboss.windup:windup-tooling shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-xml shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-base shut down by shutdown hookWeld SE container org.jboss.windup.exec:windup-exec shut down by shutdown hookWeld SE container org.jboss.forge.addon:configuration shut down by shutdown hookWeld SE container org.jboss.forge.addon:ui shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-ee shut down by shutdown hookWeld SE container org.jboss.windup.reporting:windup-reporting shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java shut down by shutdown hookIRC notifier plugin: Sending notification to: #windupSetting status of f1aae0b752655eb322a23a2f5139ed3f456c8fad to FAILURE with url http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/785/ and message: 'Build finished. 5 tests run, 0 skipped, 0 failed.'

File: reporting/impl/src/main/resources/reports/templates/migration-issues.ftl [hunk 1]
```diff
@@ -223,37 +222,7 @@
             }
         });
     });
-
-   	    function resizeTables()
-        {
```
Comment (jsight) [inline reporting/impl/src/main/resources/reports/templates/migration-issues.ftl]: Won't removing this also result in the different sections (optional vs mandatory) having differing widths?

Comment (mareknovotny) [inline reporting/impl/src/main/resources/reports/templates/migration-issues.ftl]: probably yes, I haven't noticed that before. I will re-add that js resizing

Comment (jsight) [inline reporting/impl/src/main/resources/reports/templates/migration-issues.ftl]: After testing it definitely does... but I assumed this was intentional and I guess it was just a mistake. Oops. :)

Comment (jsight) [inline reporting/impl/src/main/resources/reports/templates/migration-issues.ftl]: If this functionality gets added in, it is ready to merge, IMO.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1417/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/790/<h2>Build result: FAILURE</span></h2>[...truncated 65669 lines...][JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/classes/META-INF/resources/windup-victims-2.5.0-SNAPSHOT.dot to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT.dot[JENKINS] Archiving /var/lib/jenkins/workspace/windup-core-PR-dependents-check/windup-quickstarts/victi.ms/target/windup-victims-2.5.0-SNAPSHOT-forge-addon.jar to org.jboss.windup.quickstarts/windup-victims/2.5.0-SNAPSHOT/windup-victims-2.5.0-SNAPSHOT-forge-addon.jarchannel stoppedWeld SE container org.jboss.forge.addon:convert shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-archives shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-project shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-groovy shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-tattletale shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config-xml shut down by shutdown hookArchiving artifactsWeld SE container org.jboss.forge.addon:configuration shut down by shutdown hookWeld SE container org.jboss.windup.ui:windup-ui shut down by shutdown hookWeld SE container org.jboss.windup:windup-tooling shut down by shutdown hookWeld SE container org.jboss.windup.exec:windup-exec shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-base shut down by shutdown hookWeld SE container org.jboss.windup.ast:windup-java-ast shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-xml shut down by shutdown hookWeld SE container org.jboss.forge.addon:ui shut down by shutdown hookWeld SE container org.jboss.windup.config:windup-config shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java-ee shut down by shutdown hookWeld SE container org.jboss.windup.reporting:windup-reporting shut down by shutdown hookWeld SE container org.jboss.windup.rules.apps:windup-rules-java shut down by shutdown hookIRC notifier plugin: Sending notification to: #windupSetting status of 3ff673830422a7b63af89fe1d836efc31cbbf21e to FAILURE with url http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-core-PR-dependents-check/790/ and message: 'Build finished. 5 tests run, 0 skipped, 0 failed.'

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/795/

**Comment by mareknovotny** [comment]
I added rowspan to shown hint detail.
And when  https://issues.jboss.org/browse/WINDUP-905 show up I tried to satisfy some of the requirements there too.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/805/

**Comment by mareknovotny** [comment]
Screenshot for current status
screenshot from 2016-02-04 15-27-04

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1427/

**Comment by mareknovotny** [comment]
rebased and table columns aligned wrt WINDUP-905, but still needs to work on right side hint detail or rework it a little.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/816/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1439/<h2>Failed Tests: <span class='status-failure'>3</span></h2><h3><a name='windup-pr-builder/org.jboss.windup.rexster:windup-rexster-tests' /><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-pr-builder/1439/org.jboss.windup.rexster$windup-rexster-tests/testReport'>windup-pr-builder/org.jboss.windup.rexster:windup-rexster-tests</a>: <span class='status-failure'>1</span></h3><ul><li><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-pr-builder/1439/org.jboss.windup.rexster$windup-rexster-tests/testReport/org.jboss.windup.graph.rexster.test/RexsterTest/testRexsterProperStart/'><strong>org.jboss.windup.graph.rexster.test.RexsterTest.testRexsterProperStart</strong></a></li></ul><h3><a name='windup-pr-builder/org.jboss.windup.tests:windup-tests' /><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-pr-builder/1439/org.jboss.windup.tests$windup-tests/testReport'>windup-pr-builder/org.jboss.windup.tests:windup-tests</a>: <span class='status-failure'>2</span></h3><ul><li><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-pr-builder/1439/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureCatchallTest/testRunWindupJsp/'><strong>org.jboss.windup.tests.application.WindupArchitectureCatchallTest.testRunWindupJsp</strong></a></li><li><a href='http://wonka.mw.lab.eng.bos.redhat.com/jenkins/job/windup-pr-builder/1439/org.jboss.windup.tests$windup-tests/testReport/org.jboss.windup.tests.application/WindupArchitectureSourceModeTest/testRunWindupSourceMode/'><strong>org.jboss.windup.tests.application.WindupArchitectureSourceModeTest.testRunWindupSourceMode</strong></a></li></ul>

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-pr-builder/1440/

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://wonka.mw.lab.eng.bos.redhat.com/jenkins//job/windup-core-PR-dependents-check/818/

**Comment by mareknovotny** [comment]
closing and using https://github.com/windup/windup/pull/841 instead of this

**Pull request closed without merging**